lib/top_score_filter.py

```diff
@@ -10,6 +10,25 @@ def __init__(self, input_file, output_file, top_score_metric, file_type='pVACseq
         self.top_score_metric = top_score_metric
         self.file_type = file_type
 
+    def parse_transcript_id(self, transcript):
+        if 'ENST' in transcript:
+            return re.compile(r'ENST(\d+)').match(transcript).group(1)
+        elif 'ENSMUST' in transcript:
+            return re.compile(r'ENMUST(\d+)').match(transcript).group(1)
```

Comment:
The way the binding prediction algorithm works, the list of supported species is very small (for a complete list see https://github.com/griffithlab/pVACtools/pull/512/commits/feea7d6a6a91af9366b1fa3fa526bfca4be8001a). However, that list is probably long enough that it would make sense to use your more generalized regex.